pkg/azuredisk/controllerserver.go

```diff
@@ -186,7 +186,8 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		case fsTypeField:
 			// no op, only used in NodeStageVolume
 		case kindField:
-			// no op, only for compatibility
+			// fix csi migration issue: https://github.com/kubernetes/kubernetes/issues/103433
+			parameters[kindField] = string(v1.AzureManagedDisk)
```

Comment:
This change should to be copied to  pkg/provisioner/cloudprovisioner.go

Reply:
Ah! I modified the cloudprovisioner but didnt `git add` this change here. Thanks for pointing out. Fixed